sdk/digitaltwins/azure-digitaltwins-core/src/test/resources/session-records/componentLifcycleTest.json

```diff
@@ -1,172 +0,0 @@
-{
```

Comment:
Is this intentional?

Reply:
Yeah, the title of this file is incorrect, and it doesn't correspond to a test that we have. We still have the "componentLifecycleTest.json" file